default.nix

```diff
@@ -1,17 +1,35 @@
 let
   sources = import ./nix/sources.nix;
-  pkgs = import sources.nixpkgs {};
+  pkgs = import sources.nixpkgs {
+    config = {};
+    overlays = [
+      (import ./nix/overlay.nix)
+    ];
+  };
+  profileEnv = pkgs.writeTextFile {
+    name = "profile-env";
+    destination = "/.profile";
+    # This gets sourced by direnv. Set NIX_PATH, so `nix-shell` uses the same nixpkgs as here.
+    text = ''
+      export NIX_PATH=nixpkgs=${toString pkgs.path}
+    '';
+  };
 in {
+  inherit profileEnv;
+
   env = pkgs.buildEnv{
     name = "wire-server-deploy";
-    paths = with pkgs; [
-      terraform_0_13
-      python37Packages.poetry
-      awscli
-      sops
-      gnupg
-      yq
-      gnumake
+    paths = [
+      profileEnv
+      pkgs.ansible_2_7_with_libs
+      pkgs.awscli
+      pkgs.gnumake
+      pkgs.gnupg
+      pkgs.kubectl
+      pkgs.kubernetes-helm
```

Comment:
Can we somehow version-lock the version of helm here? Currently it's on 3.3.0 it seems.

Reply:
The version lock happens implicitly via the nixpkgs pin - we could also override the src to point to another version, but hmmh…

Is there a reason we want to explicitly stickon this helm version, instead of testing this once, while introducing, and every time we bump nixpkgs?

Reply:
I think for helm it's fine to now implicitly depend on the version provided by the nixpkgs pin